app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -99,6 +99,9 @@ def self.raw_connect(host, port, username, password, security_protocol, cert_sto
         :username => username,
         :password => password
       }
+      if cert_store.kind_of?(String)
```

Comment:
Please add line breaks before and after the block.

Reply:
Also, is there a way this can be tested? Since it broke something, I think we should have regression tests for it.

Reply:
Also, what do you think on moving this to `Endpoint` and have it default to `cert_store` if provided? I prefer to have it there, so we could remove conditionals on foreign data.

Reply:
@cfcosta linre breaks throw a rubocop error :(

Reply:
It's not the line break that throws the rubocop errors, it's the whitespaces in the blank line :)

Reply:
I clean the whitespaces and still get the same error ^^

Reply:
I was cleaning whitespaces and still got error It was confused ^^Fixed !!!